server/routerlicious/packages/lambdas/src/scriptorium/lambda.ts

```diff
@@ -57,9 +60,22 @@ export class ScriptoriumLambda implements IPartitionLambda {
 			this.providerConfig?.shouldLogInitialSuccessVerbose ?? false;
 		this.maxDbBatchSize = this.providerConfig?.maxDbBatchSize ?? 1000;
 		this.restartOnCheckpointFailure = this.providerConfig?.restartOnCheckpointFailure;
+		this.logSavedOpsTimeIntervalMs = this.providerConfig?.logSavedOpsTimeIntervalMs ?? 60000;
+		this.opsCountTelemetryEnabled = this.providerConfig?.opsCountTelemetryEnabled;
 	}
 
 	public handler(message: IQueuedMessage) {
+		if (this.opsCountTelemetryEnabled) {
+			setInterval(() => {
```

Comment:
There is an `ApiCounter` available in OSS for you to use, consider using that?

Reply:
Hey @zhangxin511 thanks for the suggestion. I looked into `ApiCounter` and yes we can implement the same logic using that as well. However, I think the current implementation also makes sense since we are counting the number of ops saved by scriptorium and not necessarily the number of calls to an api (which is already logged by Mongo api counter for insertMany - but it doesnt log how many ops were inserted). But let me know if you think strongly about using 'ApiCounter', I can update that.